tests/unit/test_transform.py

```diff
@@ -211,3 +211,145 @@ def test_transform_fields_in_json_col(self):
         )
 
         self.assertDictEqual(expected_value, return_value)
+
+    def test_transform_col_with_condition_on_json_field(self):
+        """Test transformation of a column with condition on a field in a json"""
+
+        record = {
+            "col_1": "com.transferwise.fx.user.User",
```

Comment:
nit, can we make this not W specific?

Reply:
sure

Reply:
Done https://github.com/transferwise/pipelinewise-transform-field/pull/36/commits/fcac9ffa647fd779f38ab760f211f4d178a723ab